hocr-check

```diff
@@ -57,9 +57,17 @@ def mostly_nonoverlapping(boxes,significant_overlap=0.2):
 ### main
 ################################################################
 
-if len(sys.argv)<1:
+def print_usage():
     print "usage:",sys.argv[0],"[-o] file.html"
+
+if len(sys.argv)>1 and (sys.argv[1] == '-h' or sys.argv[1] == '--help'):
+    print_usage()
+    sys.exit(0)
+
+if len(sys.argv)<1:
```

Comment:
I just did, see. e.g.https://travis-ci.org/UB-Mannheim/hocr-tools/builds/131389791 for things to fix. But I'd suggest to first fix the code until unit tests pass and then clean up the code.